### PR TITLE
publish fix

### LINKED ISSUE #164: update goggles to login3
While the main goggles webmaker nav was updated to login3, the publish dialog login system is still using the old approach, which means people cannot log in through the xray goggles publish dialog at the moment, needing to already somehow be logged in.

/cc @jbuck @cadecairos

### PR DESCRIPTION
**Pull request by Pomax**
fixes #164 -- updates the login code for the xray goggles publish dialog from old, broken login2 to new, working, login3

To test, make sure that you are running login, makeapi, and htmlsanitzer because it'll need all of those for a successful publish, and then make sure that both base login and "login only on publish" work.

The currently main landing page is still available, for adding the bookmarklet (make sure to test not just on goggles but also from bookmarklet on some random site please! O_O)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -189,7 +189,8 @@ app.get("/uproot-dialog.html", csp, function(req, res) {
     audience: env.get("audience"),
     csrf: req.csrfToken(),
     email: req.session.email || "",
-    personaHost: env.get("PERSONA_HOST")
+    personaHost: env.get("PERSONA_HOST"),
+    loginAPI: env.get("LOGINAPI")
```
Comment (jbuck) [inline app.js]: Is this being used for anything?

**Comment by jbuck** [comment]
@Pomax r+, but could you note the issue that this patch is fixing and expound on what the patch is fixing it a bit?

File: app.js (hunk 1 above)
Comment (Pomax) [inline app.js]: it's not, leftover from a `<link rel="login"...>` that got removed. pruning.

**Comment by Pomax** [comment]
The initial comment notes it's fixing 164 already. Did you want more than that?

**Pull request merged**